resolver/container.go

```diff
@@ -47,3 +47,7 @@ func (g *Container) Resolve(ctx context.Context, value string) (string, error) {
 
 	return fmt.Sprintf("%s@%s", ref.Context().Name(), resp.Digest.String()), nil
 }
+
+func (g *Container) Upgrade(ctx context.Context, value string) (string, error) {
```

Comment:
I'm not sure we want to do this? This will result in Ratchet erroring if there's a mix of container and action-based steps. I'd rather skip them and log a warning or something.